hardware/codebase/upboard/gaia/device_config.json

```diff
@@ -11,6 +11,10 @@
   "compliance": {
     "paperWallet": false
   },
+  "kioskPrinter": {
+    "maker": "Nippon",
```

Comment:
Maybe this should be ``model`` and be more specific, like ``nippon-xx-32-60mm``. What if we decide to use a different printer from Nippon in the future?